server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/DefaultOrganizationResourceMigrator.java

```diff
@@ -44,7 +43,6 @@
  * @Date: 2023/5/30 10:53
  * @Description: []
  */
-@Profile("alipay")
```

Comment:
add clientMode into @Profile rather than just remove @Profile as  some other profile has its own implementation

Reply:
done. added DesktopOrganizationResourceMigrator for clientMode only